src/platform/PlatformUtils.cc

```diff
@@ -47,6 +47,7 @@ static std::string lookupResourcesPath()
     ".", // Release location
     RESOURCE_FOLDER("../share/openscad"), // MSYS2 location
     "..", // Dev location
+    "../..", // MSVC
```

Comment:
Perhaps add that it's due to binaries are built in build/Release/ or build/Debug/